README.md

```diff
@@ -15,18 +15,21 @@ Each service might have a number of libraries available from each of the followi
 * [Management](#Management)
 
 
-### Client: July 2019 Preview
-New wave of packages that we are currently releasing in **preview**.
+### Client: November 2019 GA and Preview Releases
+New wave of packages that we are currently releasing in **preview** and several libraries are releasing in GA.
 These libraries follow the [Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet/guidelines/) and share a number of core features such as HTTP retries, logging, transport protocols, authentication protocols, etc., so that once you learn how to use these features in one client library, you will know how to use them in other client libraries. You can learn about these shared features at [Azure.Core](/sdk/core/Azure.Core/README.md).
 
 These preview libraries can be easily identified by their folder, package, and namespaces names starting with 'Azure', e.g. Azure.Storage.Blobs. 
 
-The libraries released in the July 2019 preview:
-* [Azure.ApplicationModel.Configuration](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/README.md)
-* [Azure.Messaging.EventHubs](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/README.md)
+The libraries released in the November 2019 GA release are:
```

Comment:
[Core](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/README.md) and [Identity ](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/README.md)are also missing in this release list. Should we mention these here?